Dockerfile

```diff
@@ -2,8 +2,9 @@ FROM python:3.13
 
 WORKDIR /code
 
-COPY ./pyproject.toml ./requirements.txt ./
+COPY ./pyproject.toml ./alembic.ini ./
 COPY ./src ./src
+COPY ./alembic ./alembic
```

Comment:
btw you dont need to specify ./ before each file. You are located in the root of the project by default anyway.